docs/conf.py

```diff
@@ -37,6 +37,8 @@
     'sphinx_js'
 ]
 
+js_source_path = '../fathom'
```

Comment:
Running `make docs` gives me this error:
```
Running Sphinx v2.4.4
making output directory... done
ERROR: Unable to find the source file or directory C:\cygdrive\c\Users\Daniel\code\erik-fathom\fathom

Sphinx error:
jsdoc found no JS files in the directories ['/cygdrive/c/Users/Daniel/code/erik-fathom/fathom']. Make sure js_source_path is set correctly in conf.py. It is also possible (though unlikely) that jsdoc emitted invalid JSON.
make[1]: *** [Makefile:75: html] Error 2
make[1]: Leaving directory '/cygdrive/c/Users/Daniel/code/erik-fathom/docs'
make: *** [Makefile:4: docs] Error 2
```
This is some Windows/cygwin thing that also happened before with the venv. I didn't find an easy solution yet, but I will continue looking.

Reply:
I'm curious whether running `make clean html` within the docs folder acts any differently.

Reply:
You might try `VIRTUAL_ENV ?= ./venv` instead of `VIRTUAL_ENV ?= venv` in docs/Makefile. That's the only difference I see between the docs makefile on this branch and on master.

Reply:
Of course, there's also the fact that I updated to the latest Sphinx, sphinx-js, etc., etc. Now I wish I hadn't done that.

Reply:
You know what, I didn't have the right base virtual environment when trying to build the docs the first time (I discovered this when the cli virtual environment wasn't building right) and running `make clean` does not remove the virtual environment in the docs directory so I still had a malconfigured virtual environment when trying to investigate. I manually deleted the first virtual environment and it works great now that I have the correct base virtual environment making fathom's virtual environments.